File: reverse_words/reverse_words.py
import sys

def main(file_arg):
    with open(file_arg) as f:
        for line in f:
            if line.rstrip() != '':
                split_line_reversed = line.rstrip().split(' ')[::-1]
                reversed_sentence = ' '.join(split_line_reversed)
                print(reversed_sentence)


if __name__ == '__main__':
    main(sys.argv[1])

File: reverse_words/reverse_words.js
var fs  = require("fs");

fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        line = line.replace(/\r/gm, ""); // For Windows.

        var words = line.split(" "); // Splits the string by space
        var reversedWords = [];

        for (var i = words.length - 1; i >= 0; i--) {
            var word = words[i]
            reversedWords.push(word);
        }

        console.log(reversedWords.join(' '));
    }
});